changelog/isReturnOnStack.dd

```diff
@@ -0,0 +1 @@
+Implement https://dlang.org/spec/traits.html#isReturnOnStack
```

Comment:
Please add a description, not just linking to the spec.

Reply:
There's no real point to copy/pasting. That's what links are for.

Reply:
The title is already a link. You can't put a link here. Check the preview at DAutoTest.

Also FWIW Ddoc doesn't detect URLs and you have to use LINK2...

Reply:
Ddoc does detect URLs starting with http: or https:

Reply:
DAutoTest fails:
```
.generated/stable_dmd-2.078.2/dmd2/linux/bin64/rdmd --compiler=.generated/stable_dmd-2.078.2/dmd2/linux/bin64/dmd -conf=.generated/stable_dmd-2.078.2/dmd2/linux/bin64/dmd.conf -version=Contributors_Lib ../tools/changed.d \
	 -o changelog/pending.dd --version "2.080.0" \
	--date "To be released"
object.Exception@../tools/changed.d(214): Changelog entries should consist of one title line, a blank separator line, and a description.
```

To fix this you need add a blank line and copy part of description from your spec PR.

Reply:
I'll approve this PR after this changelog entry issue is addressd.